colossalai/booster/plugin/gemini_plugin.py

```diff
@@ -62,6 +63,13 @@ def save_lr_scheduler(self, lr_scheduler: LRScheduler, checkpoint: str):
         if self.coordinator.is_master():
             super().save_lr_scheduler(lr_scheduler, checkpoint)
 
+    def save_sharded_model(self, model: GeminiDDP, checkpoint_path: str, gather_dtensor: bool = False, variant: Optional[str] = None, max_shard_size: int = 1024, use_safetensors: bool = False):
+        state_dict_shard = model.state_dict_shard(max_shard_size=max_shard_size, only_rank_0=False)
+        if self.coordinator.is_master():
+            super().save_gemini_shard_ckp(state_dict_shard, checkpoint_path, gather_dtensor, variant, use_safetensors)
```

Comment:
Why do you add this method to base class?

Reply:
the function of save_gemini_shard_ckp may be general, I'm going to rename it to save_shards.